src/panels/config/ha-panel-config.ts

```diff
@@ -345,6 +345,8 @@ class HaPanelConfig extends HassRouterPage {
 
   protected routerOptions: RouterOptions = {
     defaultPage: "dashboard",
+    beforeRender: (page) =>
+      page === "server_control" ? "../developer-tools/yaml" : undefined,
```

Comment:
Does `..` work ?

Reply:
Seems to in my testing

Reply:
We used to do this in the backend: https://github.com/home-assistant/core/blob/dev/homeassistant/components/frontend/__init__.py#L370

Reply:
Should we do it there?